core/src/main/java/com/sequenceiq/cloudbreak/core/flow2/stack/migration/AwsVariantMigrationEvent.java

```diff
@@ -0,0 +1,26 @@
+package com.sequenceiq.cloudbreak.core.flow2.stack.migration;
+
+import com.sequenceiq.cloudbreak.cloud.event.CloudPlatformResult;
+import com.sequenceiq.cloudbreak.cloud.event.resource.migration.aws.CreateResourcesResult;
+import com.sequenceiq.cloudbreak.cloud.event.resource.migration.aws.DeleteCloudFormationResult;
+import com.sequenceiq.flow.core.FlowEvent;
+
+public enum AwsVariantMigrationEvent implements FlowEvent {
```

Comment:
Shouldn't it implement IdempotentEvent interface?

Reply:
I will check this interface

Reply:
If I understood correctly, it does not need here. I'm using the `CloudStackRequest` and `CloudPlatformResult`. Unfortunately, the naming convention is not too strict here. The `FlowEvent` describe the collection of the events for the flow.